CRM/Admin/Form/AdhocChargesItem.php

```diff
@@ -131,7 +132,7 @@ public function postProcess() {
     // TODO::Make sure we cannot delete if the entity is linked to bookings
     if ($this->_action & CRM_Core_Action::DELETE) {
       CRM_Booking_BAO_AdhocChargesItem::del($this->_id);
-      CRM_Core_Session::setStatus(ts('Selected additional charges item has been deleted.'), ts('Record Deleted'), 'success');
+      CRM_Core_Session::setStatus(ts('Selected additional charges item has been deleted.'), E::ts('Record Deleted'), 'success');
```

Comment:
hey @gljk, 
Thanks for the patch, there are a few ts() which are not converted to E::ts() like on this line.
Is there a specific reason for that or can you please convert those as well?
Regards.

Reply:
ok , please can you fix these as well @gljk .
Thanks.

Reply:
@nishant-bhorodia No problem, I just did that.